### ISSUE TITLE
Failed to ping this ip

### ISSUE THREAD

**Issue by Franek96** [ORelio/Minecraft-Console-Client#489]
Console Client for MC 1.4.6 to 1.12.2 - v1.12.2 - By ORelio & Contributors
Login : franekwojc03@wp.pl
Cached session is still valid for Franek03.
Server IP : pvpwars.net
Resolving pvpwars.net...
Found server play.pvpwars.net:25565 for domain pvpwars.net
Retrieving Server Info...
A timeout occured while attempting to connect to this IP.
Failed to ping this IP.
Server version : System.Threading.ThreadAbortException: Trwało przerywanie wątku.

**Comment by Bojidar4o123** [comment]
I have the same issue

**Comment by TheSnoozer** [comment]
Could you enable debug messages by setting ``debugmessages=true`` inside your ``MinecraftClient.ini`` and try again (and post what the client is responding)?

I can remember when I had this issue what usually helped was to set a fixed version inside ``MinecraftClient.ini``  e.g. ``mcversion=1.8.0`` instead of ``mcversion=auto``

**Comment by Bojidar4o123** [comment]
I did the same thing :)

**Comment by TheSnoozer** [comment]
I just double checked that issue on my end.

If I set
```
mcversion=auto
```

it fails with the error above.

However when I set a specific version like
```
mcversion=1.8.0
```
inside the `MinecraftClient.ini` I can connect to the server without any issues.

I have seen such problems before, especially with `auto` and servers who use the a plugin like `viaversion` (not sure what plugin causes it exactly but in that area). In general what is happening here, is that during the connection the client and server negotiate what protocol to use. When using `auto` the client will use the latest protocol it can support. However then the server does not understand what the client is talking about and thus results in the above exception.

In short: the server claims to support versions, that are not supported on the server.


I'm closing this, this I'm able to connect to that server and answered. Feel free to reopen this issue, if you have a specific question to this specific problem. Else I would recommend to open a new issue.
